src/web/handlers/PanGestureHandler.ts

```diff
@@ -351,6 +361,70 @@ export default class PanGestureHandler extends GestureHandler {
     }
   }
 
+  private scheduleWheelEnd(event: AdaptedEvent) {
+    this.endWheelTimeout = setTimeout(() => {
+      if (this.receivedWheelEvent) {
+        return;
+      }
```

Comment:
Why is this needed when the timeout is cleared in `onWheel`? And why not clean it here, before scheduling a new one?

Reply:
At first I wondered if the situation below is possible:

```tsx
this.receivedWheelEvent = true;
// execute `scheduleWheelEnd` here
clearTimeout(this.endWheelTimeout);
```

but we believe that it is not possible because of how event loop works.

So yes, it is not necessary and I've removed it in [a640c0c](https://github.com/software-mansion/react-native-gesture-handler/pull/3163/commits/a640c0c7700d96d66b95ecda06b1bf8c7bd730c3)